sysdeps/sigma/meson.build

```diff
@@ -0,0 +1,40 @@
+sigma_incl = include_directories('../../subprojects/libsigma/include')
```

Comment:
Is this needed? After all, libsigma is already added as a dependency.

Reply:
I think this should be solved differently (both in the managarm sysdeps and here), as the current solution requires libsigma as a subproject but also requires it to be installed. We can merge it for now and change this later on.